app/views/migration/_form.html.erb

```diff
@@ -40,7 +40,7 @@
         <%# TODO: create a partial for web resource object fields %>
         <%= field_set_tag do %>
           <%= agg.simple_fields_for(:edm_isShownBy) do |wr| %>
-            <%= wr.input(:media, label: i18n.t('legends.each_object', number: 1), input_html: { accept: "#{EDM::WebResource.allowed_extensions}, #{EDM::WebResource.allowed_content_types}" }) %>
```

Comment:
Could you elaborate?

Reply:
The `contribution_ore_aggregation_attributes_edm_isShownBy_attributes_media` input doesn't have anything in it in the edit view. We just show the preview, but don't populate that input since the user isn't uploading a file again, and this is a file upload input.
So if you have a published story and go to the edit view via the contributions listing, you won't ever be able to submit the form since the required field will be empty.
The only way around this would be to overwrite the media by uploading a file again, or unpublishing and making changes while the media isn't required then publishing again.

Reply:
I see. Removed that then, and adjusted the migration submission system spec to suit.